src/signals/incident/components/form/MapInput/index.js

```diff
@@ -29,7 +37,7 @@ const MapInput = ({
               getError={getError}
             >
               <div className="invoer">
-                <MapInteractive onQueryResult={onQueryResult} location={value} />
+                <MapEditor onLocationChange={onLocationChange} location={value} />
```

Comment:
`onLocationChange` Doesn't seem to be a prop on the map component

Reply:
Correct, the implementation wil follow in the next pr, if I remove it here, will generate some warnings in this component that I would have to disable.